grakn-test/src/test/java/ai/grakn/test/migration/export/GraphWriterMainTest.java

```diff
@@ -33,20 +32,27 @@
     public static final EngineContext engineContext = EngineContext.startInMemoryServer();
 
     private static GraknGraph graph;
+    private String keyspace;
 
     @BeforeClass
     public static void loadMovieGraph() {
         graph =  engineContext.factoryWithNewKeyspace().getGraph();
     }
 
+    @Before
+    public void setup(){
+        keyspace = graph.getKeyspace();
+        graph.close();
+    }
```

Comment:
This looks weird to me: we open the graph only once (with `@BeforeClass`), but then we close that same graph before every test?